config_defaults_inc.php

```diff
@@ -2579,6 +2579,71 @@
 	'view_state',
 );
 
+/**
+ * An array of access levels to identify who should be presented a simplified report form.
+ *
+ * The following Statuses are standard available:
```

Comment:
No need to document it at this place.
We would have to write the same redundant paragraph to **all** options where thresholds for access levels are set.
I think It's enough what we have in Admin Guide https://mantisbt.org/docs/master/en-US/Admin_Guide/html-desktop/#admin.user.access